resources/tests/factories.py

```diff
@@ -85,8 +85,8 @@ class Meta:
 
 
 class AbstractPlaceTypeFactory(factory.django.DjangoModelFactory):
-    length = factory.Faker("random_int", min=200, max=2500)
-    width = factory.Faker("random_int", min=100, max=300)
+    length = factory.Faker("random_int", min=1, max=999)
```

Comment:
I think you meant `"pydecimal"` instead of `"random_int"`, see [this ref](https://github.com/City-of-Helsinki/berth-reservations/blob/develop/customers/tests/factories.py#L15)

Reply:
Ahhhh thanks! I looked through the Faker docs but couldn't find any.

Reply:
Faker has some very obscure docs, not an easy thing to navigate. this specific one is under ["python"](https://faker.readthedocs.io/en/latest/providers/faker.providers.python.html)